tests/ChangelogsPluginTest.php

```diff
@@ -98,8 +99,6 @@ public function test_it_is_registered_and_activated()
         $this->addComposerPlugin($plugin);
 
         $this->assertSame([$plugin], $this->composer->getPluginManager()->getPlugins());
-        $this->assertAttributeInstanceOf('Composer\IO\IOInterface', 'io', $plugin);
-        $this->assertAttributeInstanceOf('Pyrech\ComposerChangelogs\Outputter', 'outputter', $plugin);
```

Comment:
why did these 2 asserts disapear?

Reply:
Ah we may be able to add them back in again but essentially in PHPUnit 8.x `assertAttributeInstanceOf` is deprecated. Apparently the author felt that allowing tests that check the attribute value is bad practice and so removed them https://github.com/sebastianbergmann/phpunit/issues/3339

I figured here theres little to be gained by adding getters for `io` & `outputter` but can do that if you like?

Reply:
I think I added those asserts to ensure the plugin is always activated correctly (as some of its dependencies depends from Composer) : https://github.com/pyrech/composer-changelogs/blob/master/src/ChangelogsPlugin.php#L59

But anyway, we can probably drop them :wink: